### ISSUE TITLE
Ability to set Hours, Minutes, Seconds

### ISSUE THREAD

**Issue by jdvp** [maltaisn/recurpickerlib#32]
Hello,

I was wondering if you had any plan to support the Period values for Hours, Minutes, and Seconds? This would make the recurrence closer to the RFC spec.

If not, no worries, please feel free to close this issue. Just wanted to check mostly

**Comment by maltaisn** [comment]
I wasn't planning on it. Do you have specific use cases in mind?

I've been wanting to make the library closer to the RFC 5545 spec for some time now. However it's not always easy to do without sacrificing backwards compatibility. Ideally I should've used from the start a third-party library that fully implements the spec to provide the recurrence rule and recurrence finding logic. That would've saved me a headache. But I wasn't aware of the RFC spec back then. Implementing hours, minutes and seconds also implies having to think about time zones, daylight savings, leap seconds, etc. which would add a lot of complexity. (I'm already working on time zones though).

**Comment by jdvp** [comment]
@maltaisn thanks for the reply. I have a personal app that allows users to set notifications. When I started out, I also did not know about any RFCs or anything so recurrences in the app are currently only very basic (i.e. every x minutes, hours, seconds etc) and the input is just a simple edit text. 

Your solution not only looks much better but also covers more advance recurrences so I thought to ask you about the plans in case you had planned on supporting it before I wrote something custom.

Also yeah timezone/leap stuff is just about the worst thing ever. For my app, I was able to fix a lot of issues related to these by leveraging Java8 Time stuff (like ZonedDateTime) directly since AGP/Android Studio 4.0.0 supports desugaring of Java 8 code so apps can use it Java 8 Time directly now
